### ISSUE TITLE
Polygon Length doesn't update after creation

### ISSUE THREAD

**Issue by hallpell** [stevekrouse/WoofJS#606]
This code doesn't actually show the length increasing as you click on the polygon. Need to investigate further.

```
var polygonSprite1 = new Polygon({
  sides: 6, 
  length: 100, 
  color: "orange"
})


onMouseDown(() => {
  polygonSprite1.length += 100;
})
```

**Comment by hallpell** [comment]
Further breaking Polygons. Sides and length don't update dynamically when rendered, but the length is updated in the collider (but the sides are not).

```
var l = new Polygon({
  sides: 3,
  length: 50
})

l.showCollider = true

l.onMouseDown(() => {
  l.sides += 1
  l.length += 30
  l.x += 10
  new Circle({
    y: randomY()
  })
})

new Text({
  text: () => l.sides + " sides, length: " + l.length,
  y: -100,
  x: 150
})
```

This is due to code at and around https://github.com/stevekrouse/WoofJS/blob/3bc5e86950335a2f2275ebd160a78451d56c4bb6/woof.js#L1208 . The two issues I can see are (a) the setter of sides uses sides (the initial value when the polygon was constructed) as opposed to value (which we should be setting it to) at L1221 and (b) in the render function (especially L1238 and L1240), we should be using this.polygonLength and this.polygonSides as opposed to length and sides to retrieve the current value, as opposed to using the value that was fed to the constructor. Feels like it should be an easy fix.

**Comment by hallpell** [comment]
Resolved in #607, implementing the fixes discussed above.
